packages/compass-indexes/src/modules/search-indexes.ts

```diff
@@ -245,8 +250,17 @@ export default function reducer(
         isBusy: false,
       },
     };
+  } else if (
+    isAction<DropSearchIndexFailedAction>(
+      action,
+      ActionTypes.DropSearchIndexFailed
+    )
+  ) {
+    return {
+      ...state,
+      error: action.error,
```

Comment:
We show success in the toast, but error in the index view toolbar? Is this expected?

Reply:
This needs input from design. I'll check with the team.